tests/utils.py

```diff
@@ -97,7 +97,7 @@ def formatError(self, test, err):
 
 		_, module, call = test.address()
 
-		output = ['python2', 'run_tests.py', u'%s:%s' % (module, call)]
+		output = ['python2', 'run_tests.py', u'{0}:{1}'.format(module, call)]
```

Comment:
And I don't think we need `{module}:{call}`, as it's only testing code that will not get translated and it's pretty obvious, that this is just a `':'.join([module, call])`.